asf_core_data/pipeline/mcs/test/mcs_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+python gdrive_to_s3.py
+latest_outputs=$(aws s3api list-objects --bucket asf-core-data --prefix inputs/MCS/latest_raw_data/ --output text --query "Contents[].{Key: Key}")
+latest_outputs_array=($latest_outputs)
+new_installations="${latest_outputs_array[1]}"
+new_installers="${latest_outputs_array[2]}"
```

Comment:
I don't have much bash experience, but just checking that this still works now that this folder will contain multiple installation/installer files rather than just the latest ones?